extras/cloudsql/README.md

```diff
@@ -50,7 +50,7 @@ gcloud container clusters create ${CLUSTER} \
 6. **Create a Cloud SQL admin demo secret** in your GKE cluster. This gives your in-cluster Cloud SQL client a username and password to access Cloud SQL. (Note that admin/admin credentials are for demo use only and should never be used in a production environment.)
 
 ```
-INSTANCE_NAME='bank-of-anthos-db'
+INSTANCE_NAME='bank-of-anthos-db-multi'
```

Comment:
If I'm reading #507 correctly, I believe it's https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/368549b0edb4374765132fe21a87b91bcc37c295/extras/cloudsql/create_cloudsql_instance.sh#L16 that needs to change (to remove the `-multi`) rather than the other way around.
@NimJay ?

Reply:
It is already `-multi` in `extras/cloudsql-multicluster` and `extras/cloudsql/create_cloudsql_instance.sh` so I thought I would just fix the one instance.

Reply:
@bourgeoisor That's correct — we wanted to remove `-multi` from `extras/cloudsql/`.

@arueth Ah! Gotcha. I didn't realize that `extras/cloudsql/create_cloudsql_instance.sh` is being used by `extras/cloudsql-multicluster`. I, personally, prefer removing the `-multi` suffix from the Cloud SQL instance names. But I don't feel too strongly about it. So your call. :)

Reply:
@arueth I would just remove the multi from extras/cloudsql-multicluster and leave this as is. 
This script is used for a single cluster install as well so the name should not include it.